Hymns.xcodeproj/xcshareddata/xcschemes/Hymns.xcscheme

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What's this?

Reply:
I don't know exactly what this does but from what I have found online. 

From - https://stackoverflow.com/questions/53034303/should-xcshareddata-be-checked-into-git-repo-for-xcode-project

This file should be commited since it is under schareddata. "The files in xcshareddata/ should be added to the repo (but not the ones in xcuserdata/)"